modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase

```diff
@@ -1378,6 +1368,8 @@ definition {
 	@description = "LPS-147792 - Verify it is possible to see the column label as the alias on the view builder column list"
 	@priority = "5"
 	test CanSeeRenamedColumnOnViewBuilder {
+		property portal.acceptance = "true";
+		
 		ObjectAdmin.addObjectViaAPI(
 			labelName = "Custom Object 152068",
 			objectName = "CustomObject152068",
```

Comment:
This shouldn't be part of the commit `LPS-152066 Remove duplicate empty stub ` since you're adding a test to Acceptance. Please do this in a separate commit.